# ROS config
heartbeat_duration:   1.0  # secs
PID_rate:             20.0 # Hz

# Thrust controller general config
motor_cmd_limit:      2.0
neg_scale_factor:     1.9
priority_yaw_range:   1.0 # radians

# Station keeping config
#lateral_scale_x:        1.14 # experimentally found, balances strafing action
#lateral_scale_y:        2.00 # using this as scaling for lateral thruster during traversal
reconfig_duration:      1.0  # secs
station_tolerance_ang:  0.175  # rad (0.175 rad ~= 10 deg)

# Linear PID controller gains
lin_Kp: 0.25
lin_Ki: 0.0
lin_Kd: 0.1

# Angular PID controller gains
ang_Kp: 0.8
ang_Ki: 0.0
ang_Kd: 1.0

# Station PID controller gains
stn_Kp: 0.2
stn_Ki: 0.0
stn_Kd: 0.1

# General PID configuration
use_sim_time: true  # Use simulator time (true) or real time (false)
max_integral: 5.0
